jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -156,7 +152,7 @@ properties:
             title: "Date-time start of data acquisition"
             fits_keyword: DATE-BEG
             blend_table: True
-            blend_rule: mindate
+            blend_rule: mindatetime
```

Comment:
The rule mindatetime for DATE-BEG is correct, because it contains a full date+time string. DATE-END is also being populated by SDP as a full date+time string, despite the information in the keyword dictionary and the keyword comment field that suggests it is only a date. Hence the blend rule for DATE-END (line 161) needs to be changed to "maxdatetime".

Reply:
This has been remedied.